src/analysis/AnalysisNotificationManager.ts

```diff
@@ -108,6 +108,7 @@ export const AnalysisNotificationManager = (props: AnalysisNotificationManagerPr
       prevRuntime?.status !== 'Error' &&
       !_.includes(runtime.id, errorNotifiedRuntimes.get())
     ) {
+      // If you update this text, update `run-analysis-azure`
```

Comment:
thanks for adding this comment! nit, could it point to the retryUntil loop specifically?

Reply:
I want to be careful about being too specific, lest someone update the test and forget to update this comnment